### PR TITLE
Add osu! originals - osu!mania 7K World Cup 2023 news post

### PR DESCRIPTION
**Pull request by Ephemeralis**
🎹

### CODE REVIEW

File: news/2023/2023-02-15-featured-artist-updates-mwc7k.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "osu! originals - osu!mania 7K World Cup 2023"
+date: 2023-02-15 11:00:00 +0000
+---
+
+As [MWC7K](/wiki/Tournaments/MWC/2023_7K) approaches its climax, explore **5** brand new songs created specifically for the tournament!
```
Comment (Walavouchey) [inline news/2023/2023-02-15-featured-artist-updates-mwc7k.md:7]:
```suggestion
As [MWC7K](/wiki/Tournaments/MWC/2023_7K) approaches its climax, explore **5** brand-new songs created specifically for the tournament!
```

File: news/2023/2023-02-15-featured-artist-updates-mwc7k.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/fa/mwc7k2023-exclusives.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### Toromaru - Uncharted Sky
+
+This high tempo jazzy tune by [**Toromaru**](https://osu.ppy.sh/beatmaps/artists/279) isn't what you'd expect from a rhythm game, but it's exactly what you need. This acted as a *Rice* pick in the [Semifinals map pool](/wiki/Tournaments/MWC/2023_7K#semifinals.1).
```
Comment (Walavouchey) [inline news/2023/2023-02-15-featured-artist-updates-mwc7k.md:27]:
```suggestion
This high-tempo jazzy tune by [**Toromaru**](https://osu.ppy.sh/beatmaps/artists/279) isn't what you'd expect from a rhythm game, but it's exactly what you need. This acted as a *Rice* pick in the [Semifinals map pool](/wiki/Tournaments/MWC/2023_7K#semifinals.1).
```

File: news/2023/2023-02-15-featured-artist-updates-mwc7k.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+Check out [the 9-star beatmap](https://osu.ppy.sh/beatmapsets/1939418) created by [Remuring](https://osu.ppy.sh/users/6522146) and [Jinjin](https://osu.ppy.sh/users/3360737)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/273/Songs/SAMString%20-%20Ataraxia.mp3" type="audio/mpeg">
+</audio>
+
+### katagiri - Kyuukou
+
+The [Grand finals tiebreaker](/wiki/Tournaments/MWC/2023_7K#grand-finals) is a coveted position for any beatmap, and this masterpiece by [**katagiri**](https://osu.ppy.sh/beatmaps/artists/176) and its has the slot secured.
```
Comment (Walavouchey) [inline news/2023/2023-02-15-featured-artist-updates-mwc7k.md:67]:
```suggestion
The [Grand Finals tiebreaker](/wiki/Tournaments/MWC/2023_7K#grand-finals) is a coveted position for any beatmap, and this masterpiece by [**katagiri**](https://osu.ppy.sh/beatmaps/artists/176) has the slot secured.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]
"mappool" vs. "map pool"? wdyt

File: news/2023/2023-02-15-featured-artist-updates-mwc7k.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+date: 2023-02-15 11:00:00 +0000
+---
+
+As [MWC7K](/wiki/Tournaments/MWC/2023_7K) approaches its climax, explore **5** brand new songs created specifically for the tournament!
+
+![](/wiki/shared/news/banners/featured-artist.jpg)
+
+But first, some context.
+
+Our Featured Artist program has expanded over the years, and we've only recently started dipping our feet into the depths of *original song commissions* (which we call [osu! originals](/wiki/osu%21_originals)). You might've seen a few osu! originals already via Featured Artist announcements or the recent [Twin Trials contest](https://osu.ppy.sh/home/news/2023-02-01-twin-trials-contest-beatmapping-phase).
```
Comment (Walavouchey) [inline news/2023/2023-02-15-featured-artist-updates-mwc7k.md:13]:
```suggestion
Our Featured Artist program has expanded over the years, and we've only recently started dipping our feet into the depths of *original song commissions* (which we call [osu! originals](/wiki/osu!_originals)). You might've seen a few osu! originals already via Featured Artist announcements or the recent [Twin Trials contest](https://osu.ppy.sh/home/news/2023-02-01-twin-trials-contest-beatmapping-phase).
```

File: news/2023/2023-02-15-featured-artist-updates-mwc7k.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+date: 2023-02-15 11:00:00 +0000
+---
+
+As the [osu!mania World Cup 7K 2023](/wiki/Tournaments/MWC/2023_7K) approaches its climax, take a moment to explore some of the **5** brand-new songs created specifically for the tournament!
+
+![](/wiki/shared/news/banners/featured-artist.jpg)
+
+But first, some context.
+
+Our Featured Artist program has expanded over the years, and we've only recently started dipping our feet into the depths of *original song commissions* (which we call [osu! originals](/wiki/osu!_originals)). You might've seen a few osu! originals already via Featured Artist announcements or the recent [Twin Trials contest](https://osu.ppy.sh/home/news/2023-02-01-twin-trials-contest-beatmapping-phase).
```
Comment (peppy) [inline news/2023/2023-02-15-featured-artist-updates-mwc7k.md:13]: hmm, i feel like this is dismissing the originals we've had created over the years a bit by saying "only recently". some of the first featured artists were actually brought on with originals as their initial contribution (IAHN for example). we've also had some amazing landino tracks, cysmix tracks, as well as other artists over the years. not sure how to change the wording in a way that would hint at this, but maybe would be nice to?

Comment (Walavouchey) [reply news/2023/2023-02-15-featured-artist-updates-mwc7k.md:13]: this is referring to original commissions for tournaments, but i guess the wording doesn't convey that properly

Comment (Walavouchey) [reply news/2023/2023-02-15-featured-artist-updates-mwc7k.md:13]: maybe something like this

```suggestion
Our Featured Artist program has expanded over the years, and we've only recently started dipping our feet into the depths of *original tournament track commissions* (also known under the [osu! originals](/wiki/osu!_originals) label). You might've seen a few osu! originals already via Featured Artist announcements over the years, or perhaps from the recent [Twin Trials contest](https://osu.ppy.sh/home/news/2023-02-01-twin-trials-contest-beatmapping-phase).
```

**Review by Walavouchey — DISMISSED** [review]
file name needs to match title

File: news/2023/2023-02-15-featured-artist-updates-mwc7k.md (hunk 5 above)
Comment (peppy) [reply news/2023/2023-02-15-featured-artist-updates-mwc7k.md:13]: Good enough I think!

**Comment by Walavouchey** [comment]
> file name needs to match title

what i meant was that `osu! originals - osu!mania 7K World Cup 2023` does not match `2023-02-15-featured-artist-updates-mwc7k.md`

**Pull request merged**